### DISCUSSION TITLE
Response headers empty when using up:fragment:loaded event

### DISCUSSION THREAD

**Discussion by ammezie** [unpoly/unpoly#639] [General]
Hi,

I'm trying to listen to the `up:fragment:loaded` event to do a full reload on certain scenarios but I cannot access the response headers from within the event.

```
up.on('up:fragment:loaded', (event) => {
  const fullReload = event.response.header('X-Full-Reload')

  if (fullReload) {
    event.preventDefault()

    event.request.loadPage()
  }
})
```

**Comment by adam12** [comment]
Your example looks correct.

Do you receive an error? what's the error? does the server perform a redirect where the header might be lost? can you access any other headers that you know for sure exist?

**Reply by ammezie** [reply]
I have been able to resolve it. It turns out that the server isn't sending the header.
